qiskit/circuit/annotated_operation.py

```diff
@@ -150,6 +152,33 @@ def to_matrix(self):
                 raise CircuitError(f"Unknown modifier {modifier}.")
         return operator
 
+    def control(
+        self,
+        num_ctrl_qubits: int = 1,
+        label: str | None = None,
+        ctrl_state: int | str | None = None,
+        annotated: bool = False,
```

Comment:
Should this be?

```suggestion
        annotated: bool = True,
```

Not that it matters much, but we're always returning an annotated op here.

Reply:
Good point. I have also simplified the returned gate: instead of returning a control-annotated operation on top of the current annotated operation, we return an annotated operation with the current set of modifiers extended with the new control modifiers.